Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/rifleman.yml

```
@@ -136,6 +136,8 @@
 #        points: 10
       - id: CMMagazinePistolMK80
         points: 5
+      - id: RMCMagazinePistolSU6
+        points: 5
```

10 on this one as of the latest commit 4 months ago
```suggestion
        points: 10
```